src/services/PaymentService.ts

```diff
@@ -178,6 +178,16 @@ export class PaymentService {
     additionalOptions?: Partial<Stripe.SubscriptionUpdateParams>;
   }) {
     const individualActiveSubscription = await this.findIndividualActiveSubscription(customerId);
+
+    const newPrice = await this.provider.prices.retrieve(priceId, {
```

Comment:
Why is this needed?

Reply:
It's only purpose would be to validate the priceId actually belongs to an individual plan. While testing I accidentally sent a b2b priceId to an individual plan update and it basically resulted in a "lost" individual sub. I got downgraded to free. I guess this shouldn't really be an issue since the frontend wouln't specify an incorrect priceId?

Reply:
It's fine to validate these things, it's okay then